soc/arm/renesas_smartbond/da1469x/Kconfig.defconfig.series

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2022 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_DA1469X
+
+source "soc/arm/renesas_smartbond/da1469x/Kconfig.defconfig.da1469*"
+
+config SOC_SERIES
+	default "da1469x"
+
+config NUM_IRQS
+	default 40
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 32000000
```

Comment:
Why there is two places for this ?

see `soc/arm/renesas_smartbond/Kconfig.defconfig`